webapp/advantage/ua_contracts/helpers.py

```diff
@@ -0,0 +1,40 @@
+def to_dict(structure, class_key=None):
```

Comment:
I don't understand the point of this function. Could you give some examples of where you need it? What are the input structures you have and what are the output structures you're expecting?